### PR TITLE
fix: java example now works

### PR DESCRIPTION
**Pull request by jerrevanveluw**

### CODE REVIEW

**Review by wilmveel — COMMENTED** [review]
LGTM

File: examples/spring-boot-gradle-plugin/build.gradle.kts [hunk 1]
```diff
@@ -19,7 +19,6 @@ repositories {
 dependencies {
     implementation("org.springframework.boot:spring-boot-starter-web")
     implementation("org.jetbrains.kotlin:kotlin-reflect")
-    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
```
[the remark this answers is not in the document]
Comment (jerrevanveluw) [reply examples/spring-boot-gradle-plugin/build.gradle.kts:22]: jdk8 is included in stdlib as of version 1.8 I think.

File: examples/spring-boot-gradle-plugin/build.gradle.kts
Comment (ZzAve) [inline examples/spring-boot-gradle-plugin/build.gradle.kts:1]: Kunnen we hier wellicht ook spring bumpen? Ik denk dat de migratie naar SB 3 echt minimaal is, zeker gezien hier praktisch niks in zit:

https://docs.openrewrite.org/running-recipes/popular-recipe-guides/migrate-to-spring-3

Comment (jerrevanveluw) [reply examples/spring-boot-gradle-plugin/build.gradle.kts:1]: I'll do that in the next pr!

File: examples/spring-boot-maven-plugin/src/main/java/community/flock/wirespec/examples/todo_app/TodoController.java
Comment (ZzAve) [inline examples/spring-boot-maven-plugin/src/main/java/community/flock/wirespec/examples/todo_app/TodoController.java:1]: Same argument here as https://github.com/flock-community/wirespec/pull/154/files?diff=split&w=1#r1458774815, can we bump dependencies, especially Spring Boot?

**Pull request merged**